packages/paste-website/src/component-examples/ComboboxExamples.ts

```diff
@@ -324,6 +325,20 @@ render(
 )
 `.trim();
 
+export const disabledOptionsExample = `
+const products = ['SMS', 'Fax', 'Phone Numbers', 'Video', 'Email', 'Chat'];
+
+const DisabledCombobox = () => {
+  return (
+    <Combobox items={products} labelText="Select a product" disabledItems={products.slice(1,2)} />
```

Comment:
issue: this example doesn't look changed either- on the site none of the options are disabled
<img width="738" alt="image" src="https://user-images.githubusercontent.com/77117846/194105326-867b8774-d536-42c1-bf1b-bab2dfba47b3.png">
<img width="738" alt="image" src="https://user-images.githubusercontent.com/77117846/194105362-dc2dc2dc-e794-4730-863b-6e8d4d915381.png">